dashboard/app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -1,27 +1,40 @@
+<% password_reset_time = Devise.reset_password_within.inspect %>
 <% if @resource.child_users %>
+    <p><%= "#{I18n.t('devise.mailer.reset_password_instructions.hello')} #{@resource.email}" %></p>
 
-    <p>Hello <%= @resource.email %>!</p>
-
-    <p>Someone has requested a link to change the password for an account associated with your email address. You can reset the password for the appropriate account using the link below any time within the next <%= Devise.reset_password_within.inspect %>:</p>
+    <p>
+      <%= I18n.t('devise.mailer.reset_password_instructions.associated_account_password_reset') %>
+      <%= I18n.t("devise.mailer.reset_password_instructions.#{password_reset_time}") %>:
+    </p>
 
     <ul>
     <% @resource.child_users.each do |user| %>
-      <li> Change password for <%= user.username %>: <%= link_to edit_password_url(user, :reset_password_token => user.raw_token), edit_password_url(user, :reset_password_token => user.raw_token) %></li>
+      <% password_reset_url = edit_password_url(user, :reset_password_token => user.raw_token) %>
+      <li>
+        <%= "#{I18n.t('devise.mailer.reset_password_instructions.change_password_for')} #{user.username}: " %>
+        <%= link_to password_reset_url, password_reset_url %>
+      </li>
     <% end %>
     </ul>
 
-    <p>Passwords will not change until you access the corresponding link above and create a new one.</p>
+    <p><%= I18n.t('devise.mailer.reset_password_instructions.associated_account_password_wont_change') %></p>
 
 <% else %>
+    <p><%= "#{I18n.t('devise.mailer.reset_password_instructions.hello')} #{@resource.email}" %></p>
 
-    <p>Hello <%= @resource.email %>!</p>
-
-    <p>Someone has requested a link to change your password. You can do this through the link below any time within the next <%= Devise.reset_password_within.inspect %>.</p>
+    <p>
+      <%= I18n.t('devise.mailer.reset_password_instructions.your_password_reset') %>
+      <%= I18n.t("devise.mailer.reset_password_instructions.#{password_reset_time}") %>:.
```

Comment:
This seems fragile. If it changes from "7 days" to "8 days" then our password reset emails with start saying something like "Unknown string '8 days'". Is there anyway to do some datetime math, get the integer number of days, and then pass that into a string as a parameter?
```

```